docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -73,6 +73,31 @@ $ kubestellar-release
 v0.2.3-preview
 ```
 
+## Kubestellar-version
+
+This executable prints information about itself captured at build
+time.  If built by `make` then this is information conveyed by the
+Makefile; otherwise it is [the Kubernetes
+defaults](https://github.com/kubernetes/client-go/blob/master/pkg/version/base.go).
+
+It will either print one requested property to a JSON object
```

Comment:
There is a typo here, from the original but it can be fixed now.  s/to/or/

Reply:
fixed